ng2-components/ng2-alfresco-userinfo/src/components/user-info.component.html

```diff
@@ -27,7 +27,7 @@
                                      alt="ecm-profile-image"
                                      (error)="onImageLoadingError($event)"
                                      [src]="getEcmUserAvatar()"/>
-                                <h2 class="mdl-card__title-text" id="ecm-username">{{getUserNameHeaderFor('ECM')}}</h2>
+                                <h2 class="mdl-card__title-text" id="ecm-username">{{getEcmFullName()}}</h2>
```

Comment:
Instead of creating a method, the fullName should be a property of the model. In this way, it is populated just one when the model is initializing.